npp.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2020 D-Wave Systems Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+## ------- Set up our list of numbers -------
+S = [25, 7, 13, 31, 42, 17, 21, 10]
+
+## ------- Set up our QUBO dictionary -------
+
+# TODO:  Add code here to define your QUBO dictionary
+
+## ------- Run our QUBO on the QPU -------
+# TODO:  Choose QPU parameters
+chainstrength = 1
+numruns = 10
+
+# Run the QUBO on the solver from your config file
+from dwave.system import DWaveSampler, EmbeddingComposite
+
+sampler = EmbeddingComposite(DWaveSampler())
```

Comment:
Now that HSS is also online, I believe the public has access to multiple D-Wave samplers. Hence, it is necessary to filter for the QPU solver. Sorry, I should have noticed this in my earlier review.

```
EmbeddingComposite(DWaveSampler(solver={'qpu': True}))
```

Reply:
Fixed